src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1502,6 +1502,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return NewConverterInstance(assemblyQualifiedtypeName);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
DeserializedMethods.Deserialized objects shouldn't ever have any methods (unless they are primitive known types)
```powershell
Cannot bind parameter 'MemberType'. Cannot convert value "*Method*" to type "System.Management.Automation.PSMemberTypes". Error: "*Method* is not a valid value for PSMemberTypes."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1: line 19
19:         $d | Get-Member -MemberType *Method* Add | Should -BeNullOrEmpty

```